SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -312,8 +312,7 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_NON_AUTH_SERVER_INTERES
 	GSM_SHUTDOWN_COMPONENT_ID,
 
 	// Unreal load balancing components
-	VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID,
-	AUTHORITY_INTENT_COMPONENT_ID
+	VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID
```

Comment:
the auth intent component is only required by the load balancer query, and not the general server non auth interest query